address.go

```diff
@@ -25,20 +26,21 @@ type Address struct {
 	UpdatedAt          *time.Time `xml:"updated-at,omitempty"`
 }
 
+// AddressRequest represents the address used for creating payment methods.
 type AddressRequest struct {
-	XMLName            xml.Name `xml:"address"`
-	FirstName          string   `xml:"first-name,omitempty"`
-	LastName           string   `xml:"last-name,omitempty"`
-	Company            string   `xml:"company,omitempty"`
-	StreetAddress      string   `xml:"street-address,omitempty"`
-	ExtendedAddress    string   `xml:"extended-address,omitempty"`
-	Locality           string   `xml:"locality,omitempty"`
-	Region             string   `xml:"region,omitempty"`
-	PostalCode         string   `xml:"postal-code,omitempty"`
-	CountryCodeAlpha2  string   `xml:"country-code-alpha2,omitempty"`
-	CountryCodeAlpha3  string   `xml:"country-code-alpha3,omitempty"`
-	CountryCodeNumeric string   `xml:"country-code-numeric,omitempty"`
-	CountryName        string   `xml:"country-name,omitempty"`
+	XMLName            xml.Name
```

Comment:
why are we removing
```
`xml:"address"`
```
? It seems like we are always injecting it in the code?

Reply:
Yeah, that's correct. The full conversation that explains why can be followed here: https://github.com/braintree-go/braintree-go/pull/283#discussion_r272950095